### PR TITLE
grep: add Russian translation

### PR DESCRIPTION
**Pull request by dremdem**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ru/common/grep.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# grep
+
+> Поиск по шаблону в файлах используя регулярные выражения.
+> Больше информации: <https://www.gnu.org/software/grep/manual/grep.html>.
+
+- Поиск в файле по шаблону:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/grep.md:6]:
```suggestion
- Искать в файле по шаблону:
```

File: pages.ru/common/grep.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`grep "{{шаблон_поиска}}" {{путь/к/файлу}}`
+
+- Поиск по заданной подстроке (регулярные выражения отключены):
```
Comment (EmilySeville7cfg) [inline pages.ru/common/grep.md:10]:
```suggestion
- Искать по заданной подстроке (регулярные выражения отключены):
```

File: pages.ru/common/grep.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`grep --fixed-strings "{{заданная_подстрока}}" {{путь/к/файлу}}`
+
+- Поиск по шаблону во всех файлах в директории рекурсивно, показывая номера строк, там где подстрока была найдена, исключая бинарные(двоичные) файлы:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/grep.md:14]:
```suggestion
- Искать по шаблону во всех файлах директории рекурсивно, показывая номера строк, там где подстрока была найдена, исключая бинарные (двоичные) файлы:
```

File: pages.ru/common/grep.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`grep --recursive --line-number --binary-files={{without-match}} "{{шаблон_поиска}}" {{путь/к/директории}}`
+
+- Используются расширенные регулярные выражения (поддержка `?`, `+`, `{}`, `()` и `|`), без учета регистра:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/grep.md:18]:
```suggestion
 Искать, используя расширенные регулярные выражения (поддержка `?`, `+`, `{}`, `()` и `|`), без учета регистра:
```

File: pages.ru/common/grep.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Вывести 3 строки содержимого, до или после каждого совпадения:
+
+`grep --{{context|before-context|after-context}}={{3}} "{{шаблон_поиска}}" {{путь/к/файлу}}`
+
+- Вывести имя файла и номер строки для каждого совпадения:
+
+`grep --with-filename --line-number "{{шаблон_поиска}}" {{путь/к/файлу}}`
+
+- Поиск строк, где есть совпадение по шаблону поиска, вывод только совпадающей части текста:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/grep.md:30]:
```suggestion
- Искать строки, где есть совпадение по шаблону поиска, выводя только совпадающие части текста:
```

File: pages.ru/common/grep.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`grep --only-matching "{{шаблон_поиска}}" {{путь/к/файлу}}`
+
+- Поиск строк в стандартном потоке ввода которые не совпадают с шаблоном поиска:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/grep.md:34]:
```suggestion
- Искать строки в стандартном потоке ввода, которые не совпадают с шаблоном поиска:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Use [imperative mood](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood). :)

File: pages.ru/common/grep.md (hunk 1 above)
Comment (dremdem) [reply pages.ru/common/grep.md]: Changed.

File: pages.ru/common/grep.md (hunk 2 above)
Comment (dremdem) [reply pages.ru/common/grep.md]: Changed.

File: pages.ru/common/grep.md (hunk 3 above)
Comment (dremdem) [reply pages.ru/common/grep.md]: Changed.

File: pages.ru/common/grep.md (hunk 4 above)
Comment (dremdem) [reply pages.ru/common/grep.md]: Changed.

File: pages.ru/common/grep.md (hunk 5 above)
Comment (dremdem) [reply pages.ru/common/grep.md]: Changed.

File: pages.ru/common/grep.md (hunk 6 above)
Comment (dremdem) [reply pages.ru/common/grep.md]: Changed.

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**